CHANGES/829.bugfix

```diff
@@ -0,0 +1,5 @@
+Insert rescue blocks in order to flush handlers even if some tasks fail:
+Added to the following blocks: 
+  * Compile & install the pulpcore SELinux policy.
+
+Related issue : https://github.com/pulp/pulp_installer/issues/829
```

Comment:
Although our script permits this, I think the presence of multiple lines (and a list) will mess up our script.

Please just use one or more sentences on a single line.

We add the issue # automatically from the filename, no need to do that. [Example](https://docs.pulpproject.org/pulp_installer/CHANGES/)

Reply:
Yes you are right. It's something that I was wondering if it's right and if it can cause problems.
Thanks so much for the detailed explanation and support!